pdc/settings.py

```diff
@@ -93,7 +93,7 @@
     'DEFAULT_METADATA_CLASS': 'contrib.bulk_operations.metadata.BulkMetadata',
 
     'DEFAULT_FILTER_BACKENDS': ('rest_framework.filters.DjangoFilterBackend',
-                                'rest_framework.filters.OrderingFilter'),
+                                'pdc.apps.utils.utils.RelatedNestedOrderingFilter'),
```

Comment:
OK, Good idea @bliuredhat and @simozhan , will do another commit with the changes